pcsx2/USB/USB.cpp

```diff
@@ -670,6 +670,32 @@ std::string USB::GetConfigSubKey(const std::string_view& device, const std::stri
 	return fmt::format("{}_{}", device, bind_name);
 }
 
+void USB::CopyConfiguration(SettingsInterface& dest_si, const SettingsInterface& src_si)
+{
+	for (u32 port = 0; port < USB::NUM_PORTS; port++)
+	{
+		const std::string section(GetConfigSection(port));
+
+		const std::string type(GetConfigDevice(src_si, port));
+		SetConfigDevice(dest_si, port, type.c_str());
+
+		const u32 subtype(GetConfigSubType(src_si, port, type));
+
+		const gsl::span<const InputBindingInfo> bindings(GetDeviceBindings(type, subtype));
+		for (const InputBindingInfo binding : bindings)
+		{
+			const std::string binding_settings_key(GetConfigSubKey(type, binding.name));
+			dest_si.CopyStringValue(src_si, section.c_str(), binding_settings_key.c_str());
```

Comment:
Bindings are a string list, not a string (you can have more than one controller/key/etc bound to a single bind point by shift-clicking).